simtools/tests/test_simtel_config_writer.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/python3
+
+import logging
+import unittest
+
+import simtools.io_handler as io
+from simtools.simtel.simtel_config_writer import SimtelConfigWriter
+from simtools.model.telescope_model import TelescopeModel
+from simtools.layout.layout_array import LayoutArray
+
+logger = logging.getLogger()
+logger.setLevel(logging.DEBUG)
+
+
+def file_has_text(file, text):
```

Comment:
This seems to be a very generic function, useful as for other configuration writers.

Reply:
Yes, you are right. I moved it to the utils.general